dashboard/config/initializers/schema_cache_dedup.rb

```diff
@@ -138,6 +138,9 @@ def initialize(type_metadata, extra: nil, strict: false)
     private
 
     def deduplicated
+      # MySQL supports an extension for optionally specifying the display width of integer data types in parentheses.
+      # Filter display width attributes from schema-cache, they are deprecated in MySQL 8.0.17 and cause meaningless diffs.
+      @sql_type.gsub!(/\(\d+\)/, '') if @type == :integer
```

Comment:
Does type :integer also cover bigints?

Reply:
Yes- looks like [`abstract_mysql_adapter`](https://github.com/rails/rails/blob/c4d3e202e10ae627b3b9c34498afb45450652421/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L671-L675) registers all the various MySQL numeric data-types as an ActiveRecord `:integer` type.